### PR TITLE
added webpack

### PR DESCRIPTION
**Pull request by alex-zayats**
Used Babel as a loader
Used style/less and html loaders
Used ES6 modules
Used plugins
Configured 2 sets of deployment
Configured webpack-dev-server
Used AMD

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+// import '../style/style.css';
+// import loadChannels from './channels.js';
+// import loadNews from './news.js';
+// loadChannels();
```
Comment (leskeg-zz) [inline js/main.js]: Please remove commented code if not used

File: js/news.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import * as constants from './constants.js';
```
Comment (leskeg-zz) [inline js/news.js:3]: Just FYI other choice:
```javascript
import { getMoreButton, apiVersion, apiKey, articlesWrapper, getMoreButton } from './constants.js';

**Review by leskeg-zz — APPROVED** [review]
Looks really good :+1:

File: js/news.js (hunk 2 above)
Comment (alex-zayats) [reply js/news.js:3]: it's more comfortable for me, but I keep in mind your remarks

**Pull request merged**